### DISCUSSION TITLE
Gutter breaks QInput to next line

### DISCUSSION THREAD

**Discussion by eugenevk** [quasarframework/quasar#12180] [General - Components / Directives / etc] [answered]
I have a dialog on which I want to display two QInput fields on each row. I also want a gutter to create space between the two input fields. The QInputs are col-6 wide. When I don't use a gutter, two fields are nicely on one row:

![image](https://user-images.githubusercontent.com/19785727/150936703-f6347001-bf88-4f1d-a4aa-f08cbce8c20e.png)
However, when I apply q-gutter-sm to the card section, the second field moves to the next line:

![image](https://user-images.githubusercontent.com/19785727/150936828-11c0ed08-0e57-4198-8732-c45e31f04e9a.png)

This is my template:
```
<template>
  <q-page class="flex flex-center">
    <q-btn @click="dialog = true">Open dialog</q-btn>
    <q-dialog v-model="dialog">
      <q-card class="col" style="width: 400px; max-width: 500px">
        <q-card-section align="center"> Details </q-card-section>
        <q-card-section class="row q-gutter-sm">
          <q-input label="First name" class="col-6" outlined stack-label />
          <q-input label="Last name" class="col-6" outlined stack-label />
        </q-card-section>
        <q-card-actions align="center">
          <q-btn label="Close" v-close-popup />
        </q-card-actions>
      </q-card>
    </q-dialog>
  </q-page>
</template>
```
How can I have my two fields on one row with a gutter (and I don't want to use q-mr-sm for example for all fields individually)?

A codepen is here: https://codesandbox.io/s/smoosh-cherry-92k7k?file=/src/pages/Index.vue:0-660

**Comment by pdanpdan** [comment, marked as answer]
when you have children with `.col` / `.col-...` use `.q-col-gutter...`

**Reply by eugenevk** [reply]
Beautiful. Thanks a lot!
